### PR TITLE
NOISSUE - Replace swagger with openapi refs and add openapi refs

### PR DESCRIPTION
**Pull request by darkodraskovic**
Recently, in the Mainflux platform, we miggrated from swagger 2 to openapi 3 spec. We also renamed the corresponding `swagger.yml` to `openapi.yml` file. Consequently, we need to change the references to those files in the docs.

### CODE REVIEW

**Review by blokovi — DISMISSED** [review]

File: docs/storage.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ Readers provide an implementation of various `message readers`.
 Message readers are services that consume normalized (in `SenML` format) Mainflux messages from data storage and opens HTTP API for message consumption.
 Installing corresponding writer before reader is implied.
 
-Each of the Reader services exposes the same [HTTP API](https://github.com/mainflux/mainflux/blob/master/readers/swagger.yml) for fetching messages on its default port.
+Each of the Reader services exposes the same [HTTP API](https://github.com/mainflux/mainflux/blob/master/readers/openapi.yml) for fetching messages on its default port.
```
Comment (manuio) [inline docs/storage.md:78]: Can we use the same format here? `For more information about the Readers... `

Comment (darkodraskovic) [reply docs/storage.md:78]: This is just a quick fix. We need more diligence to determine where and how to refer to the HTTP API. Let's open an issue for this.

Comment (manuio) [reply docs/storage.md:78]: Maybe I was not clear. Is just proposed `For more information about Reader services API, please check out the [API documentation](https://github.com/mainflux/mainflux/blob/master/readers/openapi.yml).`

Comment (darkodraskovic) [reply docs/storage.md:78]: It's not a one solution fits all. This is just a quick fix. Please open an issue.

**Review by manuio — APPROVED** [review]

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**